Formula/openfortivpn.rb

```diff
@@ -0,0 +1,25 @@
+class Openfortivpn < Formula
+  desc "Open Fortinet client for PPP+SSL VPN tunnel services"
+  homepage "https://github.com/adrienverge/openfortivpn"
+  url "https://github.com/adrienverge/openfortivpn/archive/v1.3.1.tar.gz"
+  sha256 "cd499641e4c036cf0fc6891ce3676539c77de85a3eff9785c507ea2bd8ec3db0"
+
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "openssl"
+
+  def install
+    system "aclocal"
+    system "autoconf"
+    system "automake", "--add-missing"
+    ENV.prepend "LDFLAGS", "-L#{Formula["openssl"].opt_lib}"
+    ENV.prepend "CPPFLAGS", "-I#{Formula["openssl"].opt_include}"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+  test do
+    system bin/"openfortivpn", "--version"
```

Comment:
This is a VPN software, I've no idea on how to check its functionality without connecting to a server. --help and --version are the only informative functions provided by openfortivpn

The only tests included in the source are about code formatting, so nothing useful to check if the software works.